Automattic/jetpack
# My Jetpack: fix recommendations animation

Fixes p1733152598735019/1733149752.174589-slack-C01U2KGS2PQ

## Proposed changes:

* This PR removes the usage of `random()` from the SCSS animation so the build step doesn't generate a different CSS file every time.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






For a functionality test:
* Spin up a JN site with Jetpack Beta pointing to this feature branch
* Go to My Jetpack and activate your site with one click
* You'll be redirected to the pricing page, click to start for free or just go back to My Jetpack
* Confirm that you see a quick survey, submit the results and wait for the loading screen
* The loading screen should like the same as before
